HeroVault/HeroVault/Features/User/UserFavouritesScreen/UserFavouritesScreenViewController.swift

```diff
@@ -76,9 +76,11 @@ extension UserFavouritesScreenViewController: UICollectionViewDelegate, UICollec
             return UICollectionViewCell()
         }
         
-        let imageName = userFavouritesScreenViewModel.createImage(marvelDataIndex: indexPath.row)
-        favouritesScreenCollectionViewCell.setUpNib(imageURL: imageName)
-        favouritesScreenCollectionViewCell.layer.cornerRadius = 10
+        let marvelImage = userFavouritesScreenViewModel.createImage(marvelDataIndex: indexPath.row)
+        let marvelImageName = userFavouritesScreenViewModel.createLabel(marvelDataIndex: indexPath.row)
```

Comment:
I think you should create an image object with  ImageURL and ImageName properties
That way you can just have a single function create Image function that can return both properties 

func createImage(marvelDataIndex: indexPath.row) -> CharacterImage {}